src/ui/ImageGrid/index.tsx

```diff
@@ -1,16 +1,28 @@
 import React, { ReactElement } from 'react';
 import './index.scss';
+import { getClassName } from '../../utils';
+import { MultipleFilesMessage } from '@sendbird/chat/message';
 
 interface ImageGridProps {
   children: ReactElement[];
+  className?: string | Array<string>;
+  message: MultipleFilesMessage;
+  isReactionEnabled?: boolean;
 }
 
 export default function ImageGrid({
   children,
+  className,
+  message,
+  isReactionEnabled,
 }: ImageGridProps): ReactElement {
   return (
     <div className='sendbird-image-grid-wrap'>
-      <div className='sendbird-image-grid'>
+      <div className={getClassName([
```

Comment:
LOL, actually you don't need to accept the className as `Array<string>` type
It was my bad practice a long long time ago
Anyway this is fine too